src/components/DeeplinkWrapper/index.website.js

```diff
@@ -57,19 +51,6 @@ class DeeplinkWrapper extends PureComponent {
             this.focused = false;
         });
 
-        // check if pathname matches with deeplink routes
-        const matchedRoute = _.find(deeplinkRoutes, (route) => {
-            if (route.isDisabled && route.isDisabled(this.props.betas)) {
-                return false;
-            }
-            const routeRegex = new RegExp(route.pattern);
-            return routeRegex.test(window.location.pathname);
-        });
-
-        if (!matchedRoute) {
-            this.updateAppInstallationCheckStatus();
-            return;
-        }
```

Comment:
This caused a regression https://github.com/Expensify/App/issues/23409. After allowing all the routes to use the short lived auth token (which is triggered after the highlighted code snippet) the magic login link didn't work correctly because of the existed auth token (the one that that we attach to each url). Magic link only works if no auth token is supplied otherwise we'd be stuck on the account that we used on Web.